doc/README.rst

```diff
@@ -4,3 +4,6 @@ Style Guide <https://docs.sympy.org/dev/documentation-style-guide.html>`_.
 
 The SymPy Documentation Style Guide can also be read at
 src/documentation-style-guide.rst.
+
+Read about the official sympy docs on <https://docs.sympy.org> or on 
+<https://docs.sympy.org/latest/tutorial/index.html>
```

Comment:
Rather than "Read about" I think it would be better if this said: "The web view of the docs for the latest version is hosted at ...". I don't think there needs to be a link to the tutorial as the docs page links to the tutorial page.